doc/effects.md

```diff
@@ -65,6 +65,26 @@ square.addEffect(RotateEffect(
   curve: Curves.easeInOut,
 ));
 ```
+
+## SequenceEffect
+
+This effect is a combination of other effects. You provide it with a list of your predefined effects.
+ 
+The effects in the list should only be passed to the SequenceEffect, never added to a PositionComponent with `addEffect`.
+
+Note that no effects added to the sequence can have their `isInfinite` property set to `true`, because then naturally the sequence will get stuck once it gets to that effect.
```

Comment:
the last one could :) do we validate that or is this just a cautionary tale?

Reply:
It has an assert on if any is infininite now, but maybe we should just let the user choose if it wants to make it get stuck somewhere? And add that to the docs.

Reply:
Removed the assert and clarified the docs.